components/identity-core/org.wso2.carbon.identity.core/pom.xml

```diff
@@ -78,6 +82,11 @@
             <groupId>org.wso2.securevault</groupId>
             <artifactId>org.wso2.securevault</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.opensaml</groupId>
+            <artifactId>opensaml</artifactId>
+            <version>2.6.4</version>
```

Comment:
Don't hardcode the version. All dependency versions should be from a parent dependencyManagement section.